src/content/en/updates/2017/03/performant-expand-and-collapse.md

```diff
@@ -0,0 +1,294 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: When making expand and collapse effects you can use scale transforms to keep your UI performant.
+
+{# wf_updated_on: 2017-03-23 #}
+{# wf_published_on: 2017-03-23 #}
+{# wf_tags: performance #}
+{# wf_featured_image: /web/updates/images/2017/03/performant-expand-and-collapse/featured-image.jpg #}
+{# wf_featured_snippet: When making expand and collapse effects you can use scale transforms to keep your UI performant. #}
+
+
+# Building performant expand & collapse animations {: .page-title }
+
+{% include "web/_shared/contributors/paullewis.html" %}
+{% include "web/_shared/contributors/smcgruer.html" %}
+
+**TL;DR - use scale transforms when animating clips. You can prevent the children from being
+stretched and skewed during the animation by counter-scaling them.**
+
+Previously we’ve posted updates on how to create performant 
+[parallax effects](/web/updates/2016/12/performant-parallaxing) and 
+[infinite scrollers](/web/updates/2016/07/infinite-scroller). In this 
```

Comment:
Yes, there are going to be more, though they're not tied together in any direct way besides UI effects. I mean, we _could_ get some graphics done, though I don't know where they'd come from per se.